spidermon/contrib/validation/utils.py

```diff
@@ -28,3 +36,43 @@ def is_valid_url(url):
 
 def is_valid_email(email):
     return not EMAIL_REGEX.match(email) is None
+
+
+def get_schema_from(source):
+    if is_schema_url(source):
+        schema = get_contents(source)
+        try:
+            return json.loads(schema)
+        except Exception as e:
+            logger.exception(
+                str(e) + "\nCould not parse schema from '{}'".format(source)
+            )
+    elif source.endswith(".json"):
+        with open(source, "r") as f:
+            try:
+                return json.load(f)
+            except Exception as e:
+                logger.exception(
+                    str(e) + "\nCould not parse schema in '{}'".format(source)
+                )
+    else:
+        schema = load_object(source)
+        if isinstance(schema, six.string_types):
```

Comment:
The documentation does not mention either that the schema may be specified as a string.

I wonder if we should remove support for this, and instead allow paths to JSON files regardless of their file extension, just as we don’t require a file extension or Content-Type header when using a URL.

Reply:
From my perspective, I feel like adding raw schemas directly to the `settings.py` file as strings for validation is something users wouldn't feel like doing. Also, only my code (Cerberus Validation) has tests and docs for adding raw schemas because I thought that it would be a good feature to work with. 

Bit of context behind this change:
The code is from the original file is `/jsonschema/tools.py`, and it was moved, tweaked, and was made available for both Cerberus and JSONSchema to use. Hence, these lines are from the original file only.

Reply:
@rennerocha What are your thoughts on this?

Reply:
@rennerocha How should we proceed?

Reply:
@rennerocha pinging again. :sweat_smile:

Reply:
@rennerocha Pinging again :see_no_evil: